packages/flutter_tools/lib/src/reporting/usage.dart

```diff
@@ -72,6 +76,17 @@ class Usage {
     _analytics.setSessionValue(kSessionHostOsDetails, os.name);
     // Send the branch name as the "channel".
     _analytics.setSessionValue(kSessionChannelName, flutterVersion.getBranchName(redactUnknownBranches: true));
+    // For each flutter experimental feature, record a session value in a comma
+    // separated list.
+    final String enabledFeatures = allFeatures
+        .where((Feature feature) {
+          return feature.configSetting != null &&
+                 Config.instance.getValue(feature.configSetting) == true;
```

Comment:
umm we're missing the env vars. Should we just read straight from your _isEnabled implementation? Then we just catch all top level flutter commands. I don't think we set env vars ourselves in reentrant calls anyway.

Reply:
the env variables are going away though, these are really for our devicelab tests. we want people to use the config.

Reply:
oki

Reply:
Once we've got each of the current set enabled by default on master, I'll remove the corresponding env variable. After these four are gone I'll remove support for the feature entirely.